hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/scm/TestReconContainerManager.java

```diff
@@ -127,6 +129,33 @@ public void testCheckAndAddNewContainer() throws IOException {
         getContainerManager().getContainer(containerID).getState());
   }
 
+  @Test
+  public void testCheckAndAddNewContainerBatch() throws IOException {
+    List<ContainerReplicaProto> containerReplicaProtoList = new LinkedList<>();
+    ReconContainerManager containerManager = getContainerManager();
+    for (long i = 200L; i < 300L; i++) {
+      assertFalse(containerManager.containerExist(ContainerID.valueOf(i)));
+      ContainerReplicaProto.Builder ciBuilder =
+          ContainerReplicaProto.newBuilder();
+      ContainerReplicaProto crp = ciBuilder.
+          setContainerID(i).setState(OPEN).build();
+      containerReplicaProtoList.add(crp);
```

Comment:
Can we also testing adding containers in non OPEN state?

Reply:
added